pkg/api/mlflow/dao/repositories/helpers_test.go

```diff
@@ -84,7 +84,7 @@ func TestBuildJsonCondition(t *testing.T) {
 				"key2.nested": "value2",
 			},
 			expectedSQL:  "contexts.json->>? = ? AND contexts.json->>? = ?",
-			expectedArgs: []interface{}{"key1", "value1", "key2.nested", "value2"},
+			expectedArgs: []interface{}{"$.key1", "value1", "$.key2.nested", "value2"},
```

Comment:
For instance, this isn't the only test where we are looking at the args created by `query.go` -- there are a few cases in `query_test.go` that test the json key argument. I'm a little surprised this doesn't break more tests ... I can try out your branch, maybe the `$.` prefix is optional